package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/sdk-scheme-adapter",
-  "version": "23.6.0-iso.3",
+  "version": "23.7.0-snapshot.1",
```

Comment:
this must be restored to 23.6.0-iso.3 before merging

Reply:
Shouldn't it be done automatically after merging into `minor/iso`?

Reply:
Yes, you are right, but then we will need to update the CD to track the new range
See https://github.com/infitx-org/profile-cd/blob/main/updatecli.d/sdk-scheme-adapter.yaml

Reply:
done